homeassistant/components/idasen_desk/manifest.json

```diff
@@ -0,0 +1,16 @@
+{
+  "domain": "idasen_desk",
+  "name": "IKEA Idasen Desk",
+  "bluetooth": [
+    {
+      "service_uuid": "99fa0001-338a-1024-8a49-009c0215f78a"
+    }
+  ],
+  "codeowners": ["@abmantis"],
+  "config_flow": true,
+  "dependencies": ["bluetooth_adapters"],
+  "documentation": "https://www.home-assistant.io/integrations/idasen_desk/",
+  "iot_class": "local_push",
+  "quality_scale": "platinum",
```

Comment:
```suggestion
```

All the requirements for silver (platinum as well) aren't met here. Suggest removing it in the first PR and adding it as part of a followup as validating the platinum requirements will significantly increase review time

https://developers.home-assistant.io/docs/integration_quality_scale_index/

At least the following needs to be addressed:
> Operations like service calls and entity methods (e.g. Set HVAC Mode) have proper exception handling. Raise ValueError on invalid user input and raise HomeAssistantError for other failures such as a problem communicating with a device.
> When communicating with a device or service, the integration implements the diagnostics platform which redacts sensitive information.

Reply:
I will remove it and create a PR later with that.
Regarding "When communicating with a device or service, the integration implements the diagnostics platform which redacts sensitive information.", this does not have any sensitive info to redact, so I am not sure that applies here.

Reply:
the gold requirement for diagnostics isn't worded very well.  It needs to 

- implement diagnostics
- redact data that is sensitive as part of that

Since we only allow one platform at a time per PR, diagnostics needs to come in a future PR

Reply:
Ah I see! I need to look into that diagnostic stuff later then.